frontend/pages/journey/[id].vue

```diff
@@ -676,21 +753,135 @@ const handleUpload = (result: string) => {
                                     class="flex h-full w-full justify-center rounded-b-2xl border-x-2 border-b-2 border-natural-200 bg-natural-50 text-sm dark:border-gothic-600 dark:bg-dark"
                                 >
                                     <div
-                                        class="relative flex h-full w-full flex-col items-end"
+                                        class="relative flex h-full w-full flex-col overflow-hidden"
                                     >
-                                        <img
-                                            class="absolute right-[50%] top-[25%] z-20 w-40 -translate-y-[25%] translate-x-[50%] max-sm:mt-1"
-                                            :src="qrcode"
-                                            alt="QR Code"
-                                        />
                                         <div
-                                            class="absolute bottom-4 right-2 z-40 ml-10 flex h-16 w-16 items-center justify-center self-center rounded-full border-2 border-dashed border-natural-400 pl-1.5 pr-1.5 text-xs text-natural-400 dark:border-natural-50 dark:text-natural-50"
+                                            class="absolute bottom-4 right-2 z-40 ml-10 flex h-16 w-16 cursor-pointer select-none items-center justify-center self-center rounded-full border-2 border-dashed border-natural-400 pl-1.5 pr-1.5 text-xs text-natural-400 dark:border-natural-50 dark:text-natural-50"
                                         >
                                             <T key-name="journey.turn" />
                                         </div>
-                                        <SvgStripes
-                                            class="z-0 md:w-2/3 lg:w-1/2"
+                                        <SvgOtherStripes
```

Comment:
Can u make this line shorter please 
![image](https://github.com/user-attachments/assets/8fdb260c-426b-46bf-91d9-2a0499b5a782)